addons/danger/scripts/lambs_danger.fsm

```diff
@@ -498,7 +500,9 @@ class FSM
                                         priority = 5.000000;
                                         to="Artillery__none";
                                         precondition = /*%FSM<CONDPRECONDITION""">*/""/*%FSM</CONDPRECONDITION""">*/;
-                                        condition=/*%FSM<CONDITION""">*/"_artillery"/*%FSM</CONDITION""">*/;
+                                        condition=/*%FSM<CONDITION""">*/"_commander" \n
+                                         "&& {_vehicle getVariable [""lambs_danger_isArtillery"", getNumber (configFile >> ""CfgVehicles"" >> (typeOf _vehicle) >> ""artilleryScanner"") > 0]}" \n
```

Comment:
if we do this we maybe want to always set the variable in taskArtilleryRegister?

Reply:
Smart change!